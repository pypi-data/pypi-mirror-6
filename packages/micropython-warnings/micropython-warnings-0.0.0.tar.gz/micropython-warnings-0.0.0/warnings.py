# MicroPython doesn't support warnings
