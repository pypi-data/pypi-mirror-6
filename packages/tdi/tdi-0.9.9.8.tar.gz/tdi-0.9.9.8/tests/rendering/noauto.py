#!/usr/bin/env python
import warnings as _warnings
_warnings.resetwarnings()
_warnings.filterwarnings('error')

from tdi import html

template = html.from_string("""
<anode tdi="level1">
    <node tdi="*level2">
        <node tdi="level3">
            hey.
        </node>
    </node>
</anode>
""".lstrip())

class Model(object):
    def render_level2(self, node):
        node['foo'] = 'bar'

    def render_level3(self, node):
        node.content = 'sup.'

model = Model()
template.render(model)
