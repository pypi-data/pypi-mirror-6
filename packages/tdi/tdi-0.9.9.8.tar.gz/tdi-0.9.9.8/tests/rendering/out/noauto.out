<anode>
    <node>
        <node>sup.</node>
    </node>
</anode>
